core/interop/IDynamicInterfaceCastable/README.md

```diff
@@ -1,3 +1,15 @@
+---
+languages:
+- csharp
+- cpp
+products:
+- dotnet-core
+page_type: sample
+name: "IDynamicInterfaceCastable: Supporting Interfaces Dynamically"
+urlFragment: "idynamicinterfacecastable"
+description: "A .NET application that shows how to implement IDynamicInterfaceCastable to project a native object as implementing different managed interfaces."
+---
+
 # `IDynamicInterfaceCastable` Sample
 
 The [`IDynamicInterfaceCastable` API](https://github.com/dotnet/runtime/issues/36654) was introduced in .NET 5 as a way for creating a .NET class that supports interfaces which are not in its metadata.
```

Comment:
If this API was shipped with the current preview release, it's more reasonable to link to the api docs instead of a GitHub issue.

Reply:
I don't think the API docs have it yet - it just shipped in Preview 7 (today).

Reply:
Doc is published now - updated the link.